pages/docs/tutorials/coroutines/coroutines-basic-jvm.md

```diff
@@ -27,7 +27,7 @@ Since we'll be using the [`kotlinx.coroutines`](https://github.com/Kotlin/kotlin
 ```groovy
 dependencies {
     ...
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:{{ site.data.releases.latest.coroutines.version }}"
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:{{ site.data.releases.latest.coroutines.version }}")
```

Comment:
Please repace with multi-language sample (see how it looks [here](https://kotlinlang.org/docs/reference/using-gradle.html#configuring-dependencies))
This is the code for it:
```
<div class="multi-language-sample" data-lang="groovy">
<div class="sample" markdown="1" theme="idea" mode='groovy'>

```groovy
// groovy code
```

</div>
</div>

<div class="multi-language-sample" data-lang="kotlin">
<div class="sample" markdown="1" theme="idea" mode='kotlin' data-highlight-only>

```kotlin
// kotlin code
```

</div>
</div>

Reply:
Cool, thank you!